examples/js/postprocessing/EffectComposer.js

```diff
@@ -91,6 +92,16 @@ Object.assign( THREE.EffectComposer.prototype, {
 
 		var maskActive = false;
 
+		var currentVREnabled = this.renderer.vr.enabled;
+
+		if ( this.renderer.vr.enabled === true ) {
+
+			this.renderer.vr.enabled = false;
+			this.renderer.getDrawingBufferSize( this._rendererSize );
+			this.setSize( this._rendererSize.x, this._rendererSize.y );
```

Comment:
Calling `.setSize()` each frame might be inefficient?

Reply:
What about adding a check on `setSize()` keeping track of the last `width` and `height` used? 
```
if ( width === this.width && height === this.height ) return;
```

Reply:
That might be good.

This background of this `.setSize()` here is, `renderer` size can be changed on present change by `WebVR/XRManager`. Ideally if we can detect present change or `renderer` size change we can move out `composer` size change from `.render()`.

With WebVR API, maybe we can write like this

```javascript
var renderSize = new THREE.Vector2();

window.addEventListener( 'vrdisplaypresentchange', function ( event ) {

    renderer.getDrawingBufferSize( rendererSize );
    composer.setSize( rendererSize.x, rendererSize.y );   

}, false );
```

But I couldn't come up with an idea to do that with WebXR API. `session` event handler could do, but `session` is hidden in `WebVR.js` and `WebXRManager`.

Any ideas with WebXR API?

Reply:
Now updating composer size only when necessary.